invokeai/app/invocations/primitives.py

```diff
@@ -469,3 +470,24 @@ def invoke(self, context: InvocationContext) -> ConditioningCollectionOutput:
 
 
 # endregion
+
+# region BoundingBox
+
+
+@invocation_output("bounding_box_output")
+class BoundingBoxOutput(BaseInvocationOutput):
+    """Base class for nodes that output a single bounding box"""
+
+    bounding_box: BoundingBoxField = OutputField(description="The output bounding box.")
+
+
+@invocation_output("bounding_box_collection_output")
+class BoundingBoxCollectionOutput(BaseInvocationOutput):
+    """Base class for nodes that output a collection of bounding boxes"""
+
+    collection: list[BoundingBoxField] = OutputField(
+        description="The output bounding boxes.",
```

Comment:
Add a `title="Bounding Boxes"` to this so the node UI makes it clear what is being outputted.

Reply:
Looks like you beat me to it 👍